corehq/apps/domain/forms.py

```diff
@@ -458,10 +478,18 @@ def clean_default_geocoder_location(self):
 
     def clean_confirmation_link_expiry(self):
         data = self.cleaned_data['confirmation_link_expiry']
+        DomainGlobalSettingsForm.validate_integer_value(data, "Confirmation link expiry")
+
+    def clean_operator_call_limit(self):
+        data = self.cleaned_data['operator_call_limit']
+        DomainGlobalSettingsForm.validate_integer_value(data, "Operator call limit")
+
+    @staticmethod
+    def validate_integer_value(value, value_name):
         try:
-            return int(data)
+            return int(value)
         except ValueError:
-            raise forms.ValidationError(_("Confirmation link expiry should be an integer."))
+            raise forms.ValidationError(_(f"{value_name} should be an integer."))
```

Comment:
FYI, you can't use f strings in translations, the dynamic parts will always appear in English.

Reply:
Ok.

Reply:
@esoergel just wrote up docs explaining this: https://github.com/dimagi/commcare-hq/pull/31853